<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ButtonGroup</class>
 <widget class="QWidget" name="ButtonGroup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>28</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="buttonAdd">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>28</height>
      </size>
     </property>
     <property name="text">
      <string>新增</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonModify">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>28</height>
      </size>
     </property>
     <property name="text">
      <string>修改</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonDelete">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>28</height>
      </size>
     </property>
     <property name="text">
      <string>删除</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonSave">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>28</height>
      </size>
     </property>
     <property name="text">
      <string>保存</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonCancel">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>28</height>
      </size>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
